cheatsheets/Authentication_Cheat_Sheet.md

```diff
@@ -85,24 +85,24 @@ For more information, see: [Client-authenticated TLS handshake](https://en.wikip
 
 ## Authentication and Error Messages
 
-Incorrectly implemented error messages in the case of authentication functionality can be used for the purposes of user ID and password enumeration. An application should respond (both HTTP and HTML) in a generic manner.
+Incorrectly implemented error messages in the case of authentication functionality can be used for the purposes of the user ID and password enumeration. An application should respond (both HTTP and HTML) in a generic manner.
```

Comment:
This insertion is incorrect - it should just be `..of user ID and password enumeration`.